features/step_definitions/networking.rb

```diff
@@ -745,4 +749,29 @@
     @result = pod.exec(network_cmd, as: admin)
     raise "Failed to execute network command!" unless @result[:success]
   end
-end
\ No newline at end of file
+end
+
+Given /^I run command on the#{OPT_QUOTED} node's ovs pod:$/ do |node_name, table|
```

Comment:
is there any difference with `Given /^I run command on the#{OPT_QUOTED} node's sdn pod:$/ do |node_name, table|` step?

Reply:
@zhaozhanqi . Yes,  I need command execute failing, but  "Given /^I run command on the#{OPT_QUOTED} node's sdn pod:" has result check unless success.  I created this new one on ovs pod without result success check. So we can use this step and add success or failing check in the case.

Reply:
I did not remove that check from step "Given /^I run command on the#{OPT_QUOTED} node's sdn pod:" directly, because If this check is removed, then you need to make sure all other uses of this step are protected from failed commands as a separate step in the scenario.  So I just leave it as it is  and created this new one.

Reply:
ok. got it